src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,44 @@
+package unit.org.pih.warehouse.api
+
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import org.testng.Assert
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static io.restassured.RestAssured.*;
+import io.restassured.response.Response;
+class ProductApiTest{
+
+    Map headers = [:]
+    String baseURI = "http://localhost:8080/openboxes";
```

Comment:
I don't think if hardcoding this url is a good idea, but since I am not a good test-writer, I don't know if we can get an access to context path at this point.

Reply:
Thanks @kchelstowski I was just going to mention this.

Reply:
Note that the Rest Assured tests are executed in an isolated execution context (either as a separate process or on a separate machine) so it's sort of like running postman tests. Therefore, the API instance will be up and running before these tests are run which means we have the ability to change this URL when we execute the tests. We could even move these tests to a separate github repository to make it more clear that the tests are somewhat independent from the API. However, I don't want to do that because I want them to evolve together so we can create new tests in feature branches that live alongside the code.

I'm currently reading Testing Web APIs to see if there are any best practices to keep in mind when running these Rest Assured tests https://www.manning.com/books/testing-web-apis but I suspect that we should at least provide a default API URL in configuration that can be overridden by a system environment variable or external configuration file.